examples/example-rust-runtime/Cargo.toml

```diff
@@ -9,24 +9,30 @@ version = "0.1.0"
 # The bindings of the plugin are only required because we explicitly link against them
 # in the protocol, using `#[fp(rust_wasmer_runtime_module = "example_bindings")]` directives.
 # Normally, the runtime would never link against plugin bindings:
-example-bindings = {path = "../example-protocol/bindings/rust-plugin"}
-fp-bindgen-support = {path = "../../fp-bindgen-support", features = [
+#example-bindings = { path = "../example-protocol/bindings/rust-plugin" }
```

Comment:
So I take it we don't link to the plugin bindings anymore :) Let's just remove this line and the comment above then.